rllib/optimizers/async_replay_optimizer.py

```diff
@@ -3,15 +3,16 @@
 https://arxiv.org/abs/1803.00933"""
 
 import collections
+import logging
+import numpy as np
 import os
 import random
-import time
-import threading
-
-import numpy as np
 from six.moves import queue
+import threading
+import time
```

Comment:
Should remove the unrelated changes from this PR.

Reply:
I just reordered the imports alphabetically (my PyCharm does that).

Reply:
Ah, yes, different branch. Fixed.